packages/subgraph/TimeLock/schema.graphql

```diff
@@ -1,11 +1,24 @@
-type ProposalItem @entity {
+enum Status {
+    Pending
+    Ready
```

Comment:
suggest change `Pending` and `Ready` status to `WaitingExecuted` status.

Reply:
https://github.com/darwinia-network/wormhole-dao#proposal-lifecycle
`Pending` status could not execute the proposal